lib/ui/platform_dispatcher.dart

```diff
@@ -0,0 +1,1586 @@
+  /// [onBeginFrame]/[onDrawFrame] callback sequence or called outside the scope
+  /// of those callbacks, the call will be ignored.
+  ///
+  /// To record graphical operations, first create a [PictureRecorder], then
+  /// construct a [Canvas], passing that [PictureRecorder] to its constructor.
+  /// After issuing all the graphical operations, call the
+  /// [PictureRecorder.endRecording] function on the [PictureRecorder] to obtain
+  /// the final [Picture] that represents the issued graphical operations.
+  ///
+  /// Next, create a [SceneBuilder], and add the [Picture] to it using
+  /// [SceneBuilder.addPicture]. With the [SceneBuilder.build] method you can
+  /// then obtain a [Scene] object, which you can display to the user via this
+  /// [render] function.
+  ///
+  /// See also:
+  ///
+  ///  * [SchedulerBinding], the Flutter framework class which manages the
+  ///    scheduling of frames.
+  ///  * [RendererBinding], the Flutter framework class which manages layout and
+  ///    painting.
+  void render(Scene scene, [FlutterView view]) native 'PlatformConfiguration_render';
+
+  /// Additional accessibility features that may be enabled by the platform.
+  AccessibilityFeatures get accessibilityFeatures => configuration.accessibilityFeatures;
+
+  /// A callback that is invoked when the value of [accessibilityFeatures]
+  /// changes.
+  ///
+  /// The framework invokes this callback in the same zone in which the callback
+  /// was set.
+  VoidCallback? get onAccessibilityFeaturesChanged => _onAccessibilityFeaturesChanged;
+  VoidCallback? _onAccessibilityFeaturesChanged;
+  Zone _onAccessibilityFeaturesChangedZone = Zone.root;
+  set onAccessibilityFeaturesChanged(VoidCallback? callback) {
+    _onAccessibilityFeaturesChanged = callback;
+    _onAccessibilityFeaturesChangedZone = Zone.current;
+  }
+
+  // Called from the engine, via hooks.dart
+  void _updateAccessibilityFeatures(int values) {
+    final AccessibilityFeatures newFeatures = AccessibilityFeatures._(values);
+    final PlatformConfiguration previousConfiguration = configuration;
+    if (newFeatures == previousConfiguration.accessibilityFeatures) {
+      return;
+    }
+    _configuration = previousConfiguration.copyWith(
+      accessibilityFeatures: newFeatures,
+    );
+    _invoke(onPlatformConfigurationChanged, _onPlatformConfigurationChangedZone,);
+    _invoke(onAccessibilityFeaturesChanged, _onAccessibilityFeaturesChangedZone,);
+  }
+
+  /// Change the retained semantics data about this platform dispatcher.
+  ///
+  /// If [semanticsEnabled] is true, the user has requested that this function
+  /// be called whenever the semantic content of this platform dispatcher
+  /// changes.
+  ///
+  /// In either case, this function disposes the given update, which means the
+  /// semantics update cannot be used further.
+  void updateSemantics(SemanticsUpdate update) native 'PlatformConfiguration_updateSemantics';
```

Comment:
Will the semanticsUpdate somehow be associated with a view (i.e. are there seperate trees for each view)? Or is this still going to be one tree for everything?